src/main/java/org/folio/rest/impl/DataExportImpl.java

```diff
@@ -119,4 +119,17 @@ private void failToFetchObjectHelper(String errorMessage, Handler<AsyncResult<Re
       .onComplete(asyncResultHandler);
   }
 
+  @Override
+  public void deleteDataExportJobExecutionsById(String id, Map<String, String> okapiHeaders,
+      Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    succeededFuture().compose(ar -> jobExecutionService.deleteById(id, tenantId))
+        .map(isDeleted -> Boolean.TRUE.equals(isDeleted)
+            ? DeleteDataExportJobExecutionsByIdResponse.respond204()
+            : DeleteDataExportJobExecutionsByIdResponse
+                .respond404WithTextPlain(format("JobExecution with id '%s' was not found", id)))
+        .map(Response.class::cast).otherwise(ExceptionToResponseMapper::map)
+        .onComplete(asyncResultHandler);
```

Comment:
if it possible, format this method too, anyway, minor thing, PR looks good for me

Reply:
This is formatted according to Google Style formatter. 
https://issues.folio.org/browse/FOLIO-2127: Let's try to formalise this process